### ISSUE TITLE
Add Product Variant into Bundle

### ISSUE THREAD

**Issue by Saba-Shavidze** [Agilo/medusa-bundles#19]
Is it possible to add the product variant into the product bundle and not the product itself, or how is it possible to manage?
What I mean, I have product with different product variants, this different variants have the different prices, so for me in bundle it's important to have the actual variants in bundle not the whole product.

**Comment by HadiSDev** [comment]
Yes would be cool to instead be able to select a variant of products
